client/apps/Admin/src/components/DeleteConfirmationModal.tsx

```
@@ -0,0 +1,57 @@
+    isLoading?: boolean;
+}
+
+export const DeleteConfirmationModal: React.FC<DeleteConfirmationModalProps> = ({
+    title,
+    description,
+    onConfirm,
+    onClose,
+    isLoading = false
+}) => {
+    return (
+        <div className="delete-modal-overlay" onClick={onClose}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)